GetUsers/handler.ts

```diff
@@ -0,0 +1,153 @@
+import { ApiManagementClient } from "@azure/arm-apimanagement";
+import { Context } from "@azure/functions";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import * as express from "express";
+import { array } from "fp-ts/lib/Array";
+import { either, toError } from "fp-ts/lib/Either";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import { ServiceModel } from "io-functions-commons/dist/src/models/service";
+import {
+  AzureApiAuthMiddleware,
+  IAzureApiAuthorization,
+  UserGroup
+} from "io-functions-commons/dist/src/utils/middlewares/azure_api_auth";
+import {
+  AzureUserAttributesMiddleware,
+  IAzureUserAttributes
+} from "io-functions-commons/dist/src/utils/middlewares/azure_user_attributes";
+import {
+  ClientIp,
+  ClientIpMiddleware
+} from "io-functions-commons/dist/src/utils/middlewares/client_ip_middleware";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { withRequestMiddlewares } from "io-functions-commons/dist/src/utils/request_middleware";
+import {
+  checkSourceIpForHandler,
+  clientIPAndCidrTuple as ipTuple
+} from "io-functions-commons/dist/src/utils/source_ip_check";
+import { wrapRequestHandler } from "italia-ts-commons/lib/request_middleware";
+import {
+  IResponseErrorInternal,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+
+import { UserCollection } from "../generated/definitions/UserCollection";
+import { userContractToApiUser } from "../utils/conversions";
+import { SkipMiddleware } from "../utils/middlewares/skipMiddleware";
+
+export interface IServicePrincipalCreds {
+  readonly clientId: string;
+  readonly secret: string;
+  readonly tenantId: string;
+}
+
+export interface IAzureApimConfig {
+  readonly subscriptionId: string;
+  readonly apimResourceGroup: string;
+  readonly apim: string;
+}
+
+function getApiClient(
```

Comment:
since it is used into other methods, can you extract this method into some utils/ module ?